src/elm/Broxy.elm

```diff
@@ -38,11 +42,11 @@ type Msg
 
 init : ( Model, Cmd Msg )
 init =
-    ( { proxy = Nothing, error = Nothing }, requestISocksProxy () )
+    ( { proxy = Nothing, error = Nothing, modules = Nothing }, requestISocksProxy () )
```

Comment:
have no idea on how to subscribe for two ports in init

Reply:
@chrmod http://package.elm-lang.org/packages/elm-lang/core/5.1.1/Platform-Cmd

Commands can be batched, so using `Cmd.batch` we can provide a list of commands. That should do the trick.